src/popup/router/pages/Names/Claim.vue

```diff
@@ -27,7 +27,7 @@ export default {
   components: { NameListHeader, Input },
   data: () => ({ name: '', loading: false }),
   computed: {
-    ...mapGetters(['sdk', 'account']),
+    ...mapState(['sdk']),
     validName() {
       return this.name && /^[A-Za-z0-9]+$/.test(this.name) && this.name.length > 13;
```

Comment:
Lets extract the `validName` minimum length (this.name.length > 13) to the config/constants.